client/src/app/index.js

```diff
@@ -83,109 +84,103 @@ class App extends Component {
-                  render={(props) => (
-                    <StudentPage {...props} logout={this.logout} />
-                  )}
-                />
-
-                <Route
-                  path={ROUTES.COHORT_PROJECTS_PAGE}
-                  exact
-                  render={(props) => (
-                    <AdminProject {...props} logout={this.logout} />
-                  )}
-                />
-
-                <Route
-                  path={ROUTES.ADD_COHORT}
-                  exact
-                  render={(props) => (
-                    <AddEditForm
-                      {...props}
-                      formType="cohort"
-                      addLink="/api/v1/cohorts"
-                    />
-                  )}
-                />
-
-                <Route
-                  path={ROUTES.EDIT_COHORT}
-                  exact
-                  render={(props) => (
-                    <AddEditForm
-                      {...props}
-                      formType="cohort"
-                      editLink={`/api/v1/cohorts/${props.match.params.cohortId}`}
-                    />
-                  )}
-                />
-
-                <Route
-                  path={ROUTES.ADD_STUDENT}
-                  exact
-                  render={(props) => (
-                    <AddEditForm
-                      {...props}
-                      formType="student"
-                      addLink="/api/v1/alumni"
-                      cohortId={props.match.params.cohortId}
-                    />
-                  )}
-                />
-
-                <Route
-                  path={ROUTES.EDIT_STUDENT}
-                  exact
-                  render={(props) => (
-                    <AddEditForm
-                      {...props}
-                      formType="student"
-                      editLink={`/api/v1/alumni/${props.match.params.studentId}`}
-                      cohortId={props.match.params.cohortId}
-                    />
-                  )}
-                />
-
-                <Route
-                  path={ROUTES.ADD_PROJECT}
-                  exact
-                  render={(props) => (
-                    <AddEditForm
-                      {...props}
-                      formType="project"
-                      addLink="/api/v1/projects"
-                      cohortId={props.match.params.cohortId}
-                    />
-                  )}
-                />
-
-                <Route
-                  path={ROUTES.EDIT_PROJECT}
-                  exact
-                  render={(props) => (
-                    <AddEditForm
-                      {...props}
-                      formType="project"
-                      editLink={`/api/v1/projects/${props.match.params.projectId}`}
-                      cohortId={props.match.params.cohortId}
-                    />
-                  )}
-                />
+                <LogoutContext.Provider value={{ logout: this.logout }}>
```

Comment:
@Mu7ammadAbed I think if you wanted to, you could remove the fragment (`<>`) on line 86 because the LogoutContext now wraps all of these routes.

Reply:
Welcome back @tomduggan85 , We are so happy to have you back.

Reply:
Thanks! I'm happy to be back!